swag/swag_client.py

```diff
@@ -369,3 +416,335 @@ async def execute_swagdmin_command(self, message):
+                            participant_id, message.guild, self.client
+                        )
+                    )
+                else:
+                    participants_str.append(user.mention)
+
+            participants_mention = ", ".join(participants_str)
+
+            await message.channel.send(
+                f"{participants_mention} vous avez chacun récupéré `{gain} {cagnotte_currency}`"
+                f" de la cagnotte **{cagnotte_name}** 💸"
+            )
+
+            if gagnant_miette != None:
+                user = message.guild.get_member(gagnant_miette)
+                if user == None:
+                    user_gagnant = await get_guild_member_name(
+                        gagnant_miette, message.guild, self.client
+                    )
+                else:
+                    user_gagnant = user.mention
+                await message.channel.send(
+                    f"{user_gagnant} récupère les `{miette} {cagnotte_currency}` restants ! 🤑"
+                )
+
+            await update_forbes_classement(message.guild, self, self.client)
+
+        elif "loto" in splited_command:
+            cagnotte_idx = get_cagnotte_idx_from_command(splited_command)
+            participants_id = [participant.id for participant in message.mentions]
+
+            gagnant, gain = self.swag_bank.tirage_au_sort_cagnotte(
+                cagnotte_idx, participants_id, message.author.id
+            )
+
+            cagnotte_currency = (
+                self.swag_bank.get_active_cagnotte(cagnotte_idx)
+                .get_info()
+                .cagnotte_currency
+            )
+            cagnotte_name = (
+                self.swag_bank.get_active_cagnotte(cagnotte_idx)
+                .get_info()
+                .cagnotte_name
+            )
+
+            await message.channel.send(
+                f"{message.guild.get_member(gagnant).mention} vient de gagner l'intégralité de la €agnotte "
+                f"€{cagnotte_idx} *{cagnotte_name}*, à savoir `{gain} {cagnotte_currency}` ! 🎰"
+            )
+
+            await update_forbes_classement(message.guild, self, self.client)
+
+        elif "détruire" in splited_command:
+            cagnotte_idx = get_cagnotte_idx_from_command(splited_command)
+            cagnotte_name = (
+                self.swag_bank.get_active_cagnotte(cagnotte_idx)
+                .get_info()
+                .cagnotte_name
+            )
+
+            self.swag_bank.detruire_cagnotte(cagnotte_idx, message.author.id)
+            await message.channel.send(
+                f"La €agnotte €{cagnotte_idx} *{cagnotte_name}* est maintenant détruite de ce plan de l'existance ❌"
+            )
+            await update_forbes_classement(message.guild, self, self.client)
+
+        else:
+            await message.channel.send(
+                f"{message.author.mention}, tu as l'air perdu "
+                "(pourquoi ça ne m'étonne pas...) 🙄\nVoici les commandes "
```

Comment:
C'est aggro un peu

Reply:
Resolved in https://github.com/BobbySoftwares/Bybbot/pull/7/commits/173d6bc07cf9a1a3121d5820f16d9bcb19a74a16